.ci/install.sh

```diff
@@ -18,3 +18,5 @@ elif [[ "${BUILD_MINGW}" == ON ]]; then
 fi
 
 pip install --user --upgrade cpp-coveralls neovim
+pip-3.3 install --user --upgrade neovim
```

Comment:
I don't think its installed. Travis normally uses virtualenvs to handle python and since you don't have python set as the language I don't think the virtualenvs are set up.